doc-content/enterprise-only/processes/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
+= Filtering process instances using default or advanced filters
+
+{Central} now provides you with default and advanced filters to help you filter and search through running process instances. You can filter processes by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*. You can also create custom filters using the *Advanced Filters* option. The newly created custom filter is added to the *Saved Filters* pane, which is accessible by clicking on the star icon on the left of the *Manage Process Instances* page.
+
+[float]
+== Filtering process instances using default filters
+
+.Procedure
+. Go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* pane.
```

Comment:
Just a bit unfamiliar to me but I saw that you used _pane_, not sure if _panel_ would be more suitable.